servers/audio/effects/audio_effect_pitch_shift.cpp

```diff
@@ -293,8 +293,10 @@ void AudioEffectPitchShiftInstance::process(const AudioFrame *p_src_frames, Audi
 	float *out_l = (float *)p_dst_frames;
 	float *out_r = out_l + 1;
 
-	shift_l.PitchShift(base->pitch_scale, p_frame_count, 2048, 4, sample_rate, in_l, out_l, 2);
-	shift_r.PitchShift(base->pitch_scale, p_frame_count, 2048, 4, sample_rate, in_r, out_r, 2);
+	static const int fft_sizes[AudioEffectPitchShift::FFT_Size::FFT_SIZE_MAX] = { 256, 512, 1024, 2048, 4096 };
```

Comment:
GCC 5 seems to fail on this:
```
servers/audio/effects/audio_effect_pitch_shift.cpp: In member function 'virtual void AudioEffectPitchShiftInstance::process(const AudioFrame*, AudioFrame*, int)':
servers/audio/effects/audio_effect_pitch_shift.cpp:296:52: error: 'AudioEffectPitchShift::FFT_Size' is not a class or namespace
  static const int fft_sizes[AudioEffectPitchShift::FFT_Size::FFT_SIZE_MAX] = { 
                                                    ^
```
I think you should be able to use `FFT_SIZE_MAX` directly here.